test/asv.linux.conf.json

```diff
@@ -34,20 +34,20 @@
         "python -c \"import shutil; shutil.rmtree('{build_dir}/qiskit', True)\"",
         "python -c \"import shutil; shutil.rmtree('{build_dir}/qiskit_aer.egg-info', True)\"",
         "pip install git+https://github.com/Qiskit/qiskit-terra",
-        "pip install git+https://github.com/Qiskit/qiskit-aqua",
+        "pip install qiskit_nature",
```

Comment:
How come we install Terra (and previously Aqua) from GitHub, but Nature from pip?

Reply:
sometimes qiskit-aer has dependency from the main of terra and will not work well with the pip-latest version.